README.md

```diff
@@ -20,12 +26,276 @@ $ python setup.py install # install wallet-sdk-py
+...         "public_key_data": ""
+...     }
+... }
+>>> _, resp = wallet.register(header, body)
+>>> print resp
+```
+
+* `Callback-Url` in http header is optional. If you want to receive blockchain transaction
+event, you need to set `Callback-Url` in header; if not, you don't have to set it.
+
+### Create POE digital asset
+
+After creation wallet account, you can create POE asset for this account as follows:
+
+```python
+>>> from api.poe import POEClient
+>>> poe = POEClient(url, apikey, cert_path)
+>>> creator = "wallet ID"
+>>> created = "123456" ## timestamp when poe is created
+>>> privateB64 = "base64 formatted private key"
+>>> payload = {
+...     "id": "",
+...     "name": "name",
+...     "parent_id": "parent-poe-id",
+...     "owner": "owner did",
+...     "hash": "metadata-hash",
+...     "metadata": '{
+...         "address": "xxx",
+...         "telephone": "xxx",
+...         ...
+...     }'
+... }
+>>> params = {
+...     "header": {},
+...     "creator": creator,
+...     "created": created,
+...     "privateB64": privateB64,
+...     "payload": payload
+...     }
+>>> _, resp = poe.create_with_sign(**params)
+>>> print resp
+```
+
+* At creating POE asset, the **name** and **owner** fields must be set, and the
+**owner** field must be set to the wallet account ID.
+
+* At building the signature parameter, using the ed25519 private key returned
+by regiser wallet API to do ed25519 signature.
+
+### Upload POE file
+
+After creation POE, you can upload POE file for this account as follows:
+
+```python
+>>> filename = "file path"
+>>> poeid = "poeid"
+>>> _, resp = poe.upload({}, filename, poeid)
+>>> print resp
+```
+
+* `upload` API uploads the file to **Offchain** storage, and generates SHA256
+hash value for this file, then saves this hash value into blockchain.
+
+### Issue colored token using digital asset
+
+Once you have specified asset, you can use this specified asset to issue colored
+token as follows:
+
+```python
+>>> from api.transaction import Transaction
+>>> txn = Transaction(url, apikey, cert_path)
+>>> creator = "creator"
+>>> created = 12345
+>>> privateB64 = "base64 formatted private key"
+>>> payload = {
+...     "issuer": "issuer did",
+...     "owner": "owner did",
+...     "asset_id": "asset did",
+...     "amount": 1000,
+...     "fees": {
+...         "accounts": [
+...             "accounts did"
+...         ],
+...         "coins": [
+...             {
+...                 "coin_id": "coin id",
+...                 "amount": 5
+...             }
+...         ]
+...     }
+... }
+>>> params = {
+...     "header": {},
+...     "creator": creator,
+...     "created": created,
+...     "privateB64": privateB64,
+...     "payload": payload
+...     }
+>>> _, resp = txn.issue_colored_token_with_sign(**params)
+```
+
+* At issuing colored token, you need to specify an issuer(one wallet account ID),
+an asset to be used to issue token, and the owner(another wallet account ID) who
```

Comment:
an asset to issue tokens, and the owner (...).

(owner already means the one who has the asset)